app/sidekiq/lighthouse/form526_document_upload_polling_job.rb

```diff
@@ -41,32 +44,49 @@ class Form526DocumentUploadPollingJob
     end
 
     def perform
-      Lighthouse526DocumentUpload.pending.status_update_required.in_batches(
+      successful_documents_before_polling = Lighthouse526DocumentUpload.completed.count
+      failed_documents_before_polling = Lighthouse526DocumentUpload.failed.count
+
+      documents_to_poll = Lighthouse526DocumentUpload.pending.status_update_required
+      StatsD.gauge("#{STATSD_KEY_PREFIX}.#{STATSD_PENDING_DOCUMENTS_POLLED_KEY}", documents_to_poll.count)
+
+      documents_to_poll.in_batches(
         of: POLLED_BATCH_DOCUMENT_COUNT
       ) do |document_batch|
         lighthouse_document_request_ids = document_batch.pluck(:lighthouse_document_request_id)
-        response = BenefitsDocuments::Form526::DocumentsStatusPollingService.call(lighthouse_document_request_ids)
-
-        if response.status == 200
-          result = BenefitsDocuments::Form526::UpdateDocumentsStatusService.call(document_batch, response.body)
-
-          if result && !result[:success]
-            response_struct = OpenStruct.new(result[:response])
 
-            handle_error(response_struct, response_struct.unknown_ids.map(&:to_s))
-          end
-        else
-          handle_error(response, lighthouse_document_request_ids)
-        end
+        update_document_batch(document_batch, lighthouse_document_request_ids)
       rescue Faraday::ResourceNotFound => e
         response_struct = OpenStruct.new(e.response)
 
         handle_error(response_struct, lighthouse_document_request_ids)
       end
+
+      documents_marked_success = Lighthouse526DocumentUpload.completed.count - successful_documents_before_polling
+      StatsD.gauge("#{STATSD_KEY_PREFIX}.#{STATSD_PENDING_DOCUMENTS_MARKED_SUCCESS_KEY}", documents_marked_success)
+
+      documents_marked_failed = Lighthouse526DocumentUpload.failed.count - failed_documents_before_polling
+      StatsD.gauge("#{STATSD_KEY_PREFIX}.#{STATSD_PENDING_DOCUMENTS_MARKED_FAILED_KEY}", documents_marked_failed)
```

Comment:
Any appetite to add a `# not marked` stat just incase? 

IE
```
documents_to_poll.count - documents_marked_success - documents_marked_failed
```

Reply:
eh... we dont need to, we have the others all in metrics, can do it DD side via a metric formula, no need